Planka: update command fails

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Planka

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/planka.sh)", after creation update

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 📝 Provide a clear and concise description of the issue.

`update` command fails after creating a new lxc with the newest planka script.

### 🔄 Steps to reproduce the issue.

Execute `bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/planka.sh)"` in proxmox root shell
after creation log into lxc and execute `update`

### ❌ Paste the full error output (if available).

Updating PLANKA to v2.0.0-rc.3mv: cannot stat '/opt/planka/public/favicons/*': No such file or directory

[ERROR] in line 44: exit code 0: while executing command mv /opt/planka/public/favicons/* /opt/planka-backup/favicons/

### 🖼️ Additional context (optional).

<img width="1082" height="270" alt="Image" src="https://github.com/user-attachments/assets/7b3f04aa-f19a-4548-abb9-debe511248cb" />

--- a/ct/planka.sh
+++ b/ct/planka.sh
@@ -35,25 +35,21 @@ function update_script() {
     msg_ok "Stopped $APP"
 
     msg_info "Updating $APP to ${RELEASE}"
-    mkdir -p /opt/planka-backup
-    mkdir -p /opt/planka-backup/favicons
-    mkdir -p /opt/planka-backup/user-avatars
-    mkdir -p /opt/planka-backup/background-images
-    mkdir -p /opt/planka-backup/attachments
+    mkdir -p /opt/planka-backup/{favicons,user-avatars,background-images,attachments}
     mv /opt/planka/.env /opt/planka-backup
-    [ -n "$(ls -A /opt/planka/public/favicons 2>/dev/null)" ] && mv /opt/planka/public/favicons/* /opt/planka-backup/favicons/
-    [ -n "$(ls -A /opt/planka/public/user-avatars 2>/dev/null)" ] && mv /opt/planka/public/user-avatars/* /opt/planka-backup/user-avatars/
-    [ -n "$(ls -A /opt/planka/public/background-images 2>/dev/null)" ] && mv /opt/planka/public/background-images/* /opt/planka-backup/background-images/
-    [ -n "$(ls -A /opt/planka/private/attachments 2>/dev/null)" ] && mv /opt/planka/private/attachments/* /opt/planka-backup/attachments/
+    [ -d /opt/planka/public/favicons ] && [ "$(ls -A /opt/planka/public/favicons)" ] && mv /opt/planka/public/favicons/* /opt/planka-backup/favicons/
+    [ -d /opt/planka/public/user-avatars ] && [ "$(ls -A /opt/planka/public/user-avatars)" ] && mv /opt/planka/public/user-avatars/* /opt/planka-backup/user-avatars/
+    [ -d /opt/planka/public/background-images ] && [ "$(ls -A /opt/planka/public/background-images)" ] && mv /opt/planka/public/background-images/* /opt/planka-backup/background-images/
+    [ -d /opt/planka/private/attachments ] && [ "$(ls -A /opt/planka/private/attachments)" ] && mv /opt/planka/private/attachments/* /opt/planka-backup/attachments/
     rm -rf /opt/planka
     fetch_and_deploy_gh_release "planka" "plankanban/planka" "prebuild" "latest" "/opt/planka" "planka-prebuild.zip"
     cd /opt/planka
     $STD npm install
     mv /opt/planka-backup/.env /opt/planka/
-    [ -n "$(ls -A /opt/planka-backup/favicons 2>/dev/null)" ] && mv /opt/planka-backup/favicons/* /opt/planka/public/favicons/
-    [ -n "$(ls -A /opt/planka-backup/user-avatars 2>/dev/null)" ] && mv /opt/planka-backup/user-avatars/* /opt/planka/public/user-avatars/
-    [ -n "$(ls -A /opt/planka-backup/background-images 2>/dev/null)" ] && mv /opt/planka-backup/background-images/* /opt/planka/public/background-images/
-    [ -n "$(ls -A /opt/planka-backup/attachments 2>/dev/null)" ] && mv /opt/planka-backup/attachments/* /opt/planka/private/attachments/
+    [ -d /opt/planka-backup/favicons ] && [ "$(ls -A /opt/planka-backup/favicons)" ] && mv /opt/planka-backup/favicons/* /opt/planka/public/favicons/
+    [ -d /opt/planka-backup/user-avatars ] && [ "$(ls -A /opt/planka-backup/user-avatars)" ] && mv /opt/planka-backup/user-avatars/* /opt/planka/public/user-avatars/
+    [ -d /opt/planka-backup/background-images ] && [ "$(ls -A /opt/planka-backup/background-images)" ] && mv /opt/planka-backup/background-images/* /opt/planka/public/background-images/
+    [ -d /opt/planka-backup/attachments ] && [ "$(ls -A /opt/planka-backup/attachments)" ] && mv /opt/planka-backup/attachments/* /opt/planka/private/attachments/
     msg_ok "Updated $APP to ${RELEASE}"
 
     msg_info "Starting $APP"

